src/core/vdom.ts

```diff
@@ -813,7 +813,7 @@ export function renderer(renderer: () => RenderResult): Renderer {
 		merge: true,
 		transition: transitionStrategy,
 		domNode: global.document.body,
-		registry: null
+		registry: new Registry()
```

Comment:
Add a `Registry` by default to the application if one is not passed by the user - this means things like the `theme` middleware will work without passing a registry to the `mount`